tsconfig.json

```diff
@@ -2,13 +2,14 @@
   "compilerOptions": {
     "target": "es2015",
     "module": "commonjs",
-    "declaration": true,
+    // "declaration": true,
```

Comment:
Could be added back once we have included a fixed mongoose-findorcreate

Reply:
See my comment below